docs/redis.md

```diff
@@ -49,6 +49,7 @@ The message body should be a JSON object with the following fields:
 
 - `input`: a JSON object with the same keys as the [arguments to the `predict()` function](python.md). Any `File` or `Path` inputs are passed as URLs.
 - `webhook`: the URL Cog will send responses to.
+- `cancel_key` (optional): a Redis key to watch to signal that this prediction should be canceled. If the key is created the prediction will be canceled.
```

Comment:
Hmm – not sure about this API because it's Redis specific but in a generic top-level key on the prediction object.

I've made a note here and we can iterate before we ship the next version of cog: https://github.com/replicate/replicate-web/issues/2146#issuecomment-1269706369